.github/workflows/size.yml

```diff
@@ -0,0 +1,104 @@
+name: Size
+
+# read-write repo token
+# access to secrets
+#
+# https://securitylab.github.com/resources/github-actions-preventing-pwn-requests/
+on:
+  workflow_run:
```

Comment:
I would make this workflow run on `pull_request` and save `out.txt` from the PR to artifacts. We could then add another workflow to run the sizes on `main` and keep that in an artifact as well to save some CI time.

The other workflow, `comment-sizes-pr` or similar, would use both outputs to comment on the PR.

Reply:
done (tho I think you have the workflows backwards :upside_down_face: )